desktop/cmp/rest/RestGridModel.js

```diff
@@ -12,6 +12,53 @@ import {Icon} from '@xh/hoist/icon';
 import {pluralize} from '@xh/hoist/utils/js';
 
 import {RestFormModel} from './RestFormModel';
+import {withDefault} from '../../../utils/js/LangUtils';
+
+export const restGridAddAction = {
```

Comment:
Very cool -- wonder if we need the 'restGrid' prefix here.  Its implied by package we are importing from